src/main/java/com/powsybl/network/conversion/server/NetworkConversionController.java

```diff
@@ -100,4 +100,12 @@ public ResponseEntity<NetworkInfos> importCgmesCase(@RequestParam("caseUuid") UU
         var networkInfos = networkConversionService.importCgmesCase(caseUuid, boundaries);
         return ResponseEntity.ok().body(networkInfos);
     }
+
+    @PostMapping(value = "/networks/{networkUuid}/reindex-all")
```

Comment:
Maybe use POST /networks/{networkUuid}/indexes ?

Reply:
I used the API defined in the US

Reply:
Ok